core/app/[locale]/(default)/account/(tabs)/settings/_components/change-password-form.tsx

```diff
@@ -107,14 +108,20 @@ export const ChangePasswordForm = () => {
   const [isNewPasswordValid, setIsNewPasswordValid] = useState(true);
   const [isConfirmPasswordValid, setIsConfirmPasswordValid] = useState(true);
 
+  const { setAccountState } = useAccountStatusContext();
+
   useEffect(() => {
     if (state.status === 'success') {
-      setTimeout(() => {
-        void logout();
-        router.push('/login');
-      }, 2000);
+      void logout();
+
+      setAccountState({
+        status: 'success',
+        message: t('confirmChangePassword'),
+      });
+
+      router.push('/login');
```

Comment:
`logout` should redirect to login page, no?

Reply:
I made some changes in the [logout](core/components/header/_actions/logout.ts) method, but now if the user decides to log out from any page, he will automatically be redirected to the Login page. Perhaps it is worth clarifying this behavior with Turner, because now when logging out, the user remains on the same page.

Otherwise, if we don't change the `logout` method, we can use `router.push('/login')` after successfully changing password.